pyprep/find_noisy_channels.py

```diff
@@ -586,19 +599,17 @@ def get_ransac_pred(self, chn_pos, chn_pos_good, good_chn_labs, n_pred_chns, dat
             channel numbers used for interpolation for RANSAC
         data : ndarray
             2-D EEG data
+        sample: int
```

Comment:
in numpydoc style, there is a space before the `:`

```suggestion
        sample : int
```